src/components/screen/MemoList/MemoList.tsx

```diff
@@ -8,17 +8,17 @@ const MemoList: React.FC<MemoListProps> = ({ users }) => {
     })
 
   return (
-    <div className='w-11/12 gap-10 rounded-lg border bg-white p-4 md:w-4/5 md:p-8'>
-      <p className='text-xl'>メモ欄</p>
+    <div className='w-11/12 gap-10 rounded-lg border bg-white  p-4 drop-shadow-md md:w-4/5 md:p-8'>
+      <p className='text-xl'>コメント欄</p>
       <div className='divider' />
       <div className='chat chat-start items-end gap-4'>
         {filterUsers &&
           filterUsers.map((user) => (
             <>
-              <p className='rounded-2xl bg-primary p-2 text-lg text-white md:ml-4 md:p-3'>
+              <p className='rounded-box border p-2 text-lg text-primary outline md:ml-4 md:p-3'>
                 {user.name}
               </p>
-              <div className='chat-bubble chat-bubble-secondary my-auto'>
+              <div className='chat-bubble chat-bubble-primary my-auto'>
                 <p className='hidden-scrollbar overflow-auto whitespace-nowrap break-keep text-xs text-white md:text-sm'>
                   {user.memo}
                 </p>
```

Comment:
```suggestion
              <p className='rounded-box borderp-2 text-lg text-primary outline md:ml-4 md:px-3 md:py-1'>
                {user.name}
              </p>
              <div className='chat-bubble chat-bubble-primary h-auto flex items-center' >
                <p className='hidden-scrollbar overflow-x-auto whitespace-nowrap text-xs text-white'>
                  {user.memo}
                </p>
              </div>
```
こんな感じに書いたらtextが中央寄せになる！